bins/tool.mjs

```diff
@@ -14,4 +14,10 @@ config({
 
 process.env.BUILD_OUT_ROOT = process.env.BUILD_OUT_ROOT || BUILD_OUT_ROOT;
 
-spawn(rariBin, ["content", ...process.argv.slice(2)], { stdio: "inherit" });
+const child = spawn(rariBin, ["content", ...process.argv.slice(2)], {
+  stdio: "inherit",
+});
+
+child.on("close", (code) => {
+  process.exit(code);
```

Comment:
```suggestion
  process.exitCode = code;
```

Form the NodeJS docs:
> Rather than calling process.exit() directly, the code should set the process.exitCode and allow the process to exit naturally by avoiding scheduling any additional work for the event loop: